notagain/lua/notagain/pac_server/autorun/dev_helpers.lua

```diff
@@ -0,0 +1,26 @@
+local Tag = "Say"
+
+if SERVER then
+	util.AddNetworkString(Tag)
+	local META = FindMetaTable('Player');
+	function META:Say(string)
+		net.Start(Tag)
+			net.WriteString(string)
+		net.Send(self)
+	end
+end
+
+if CLIENT then
+	net.Receive(Tag, function(len, ply)
+		Say(net.ReadString())
+	end)
+end
+
+function Say(string)
+	string.Replace( string, '"', '\"' )
+	if CLIENT then
```

Comment:
no it isn't, checking once when the file loads is best than everytime you try to use the function